### ISSUE TITLE
Incorrect port number in uris of getinfo

### ISSUE THREAD

**Issue by Kixunil** [lightningnetwork/lnd#4736]
### Background

The port present in `uris` field returned by `getinfo` is not the same as set via `listen`. It's 9735 even if `listen` contains a different port.

### Your environment

* version of `lnd` 0.11.1-beta
* which operating system (`uname -a` on *Nix) `Linux <censored> 4.19.0-10-amd64 #1 SMP Debian 4.19.132-1 (2020-07-24) x86_64 GNU/Linux`
* version of `btcd`, `bitcoind`, or other backend `Bitcoin Core version v0.20.0`
* any other relevant environment details

### Steps to reproduce

1. setup LND with `listen=0.0.0.0:9736` and `externalip=YOUR_IP_OR_ONION`
2. run `lncli getinfo`

### Expected behaviour

Returned `uris` field contains port `9736`

### Actual behaviour

Returned `uris` field contains port `9735`

**Comment by wpaulino** [comment]
The two config options aren't linked together at the moment. `externalip` allows you to set a port as well, so if you're listening on a different port than 9735 externally, you'll need to specify it there as well. We could detect that if `externalip` doesn't have a port then we use whatever we're listening on if you'd like to propose a PR to do so.

**Comment by Kixunil** [comment]
Yeah, that'd be helpful. I don't have much experience with Go, so I can't promise a PR any time soon. I will fix it on my side (which is trivial) for the time being.

**Comment by saubyk** [comment]
Haven't head user complaints about this issue, closing unless there is an appetite from someone to submit a pr addressing this.
